src/main/java/org/medicmobile/webapp/mobile/UrlHandler.java

```diff
@@ -17,10 +17,13 @@
 import android.webkit.WebView;
 import android.webkit.WebViewClient;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
 public class UrlHandler extends WebViewClient {
 	EmbeddedBrowserActivity parentActivity;
 	SettingsStore settings;
 
+	@SuppressFBWarnings("EI_EXPOSE_REP2")
```

Comment:
We're suppressing this a lot. Maybe we should just ignore it throughout rather than add all these annotations?

Reply:
Done, adding a note saying that we omit this rule because we connect with external apps and assume the data exchanged is safe.